### ISSUE TITLE
Kfctl create profile for user by default

### ISSUE THREAD

**Issue by jlewi** [kubeflow/kubeflow#3639]
With 0.6.0 a user can no longer create notebooks in the "kubeflow" namespace. They need to create a profile (see https://bit.ly/2UOzzxB) which will create the namespace with appropriate service accounts.

```
apiVersion: kubeflow.org/v1alpha1
kind: Profile
metadata:
  name: <name of the name space to be used>
spec:
  owner:
    kind: User
    name: <ID of the user to use it>
```

We are planning a UI based flow to do that but the UI hasn't landed yet.

In the meantime, we could have `kfctl` automatically create profile for ${USER} running kfctl.
This would mean out of the box the user has a namespace they can run Kubeflow workloads in.

@yanniszark @kkasravi WDYT? Are either of you able to pick this up?

**Comment by yanniszark** [comment]
@jlewi I can take this one.
/assign @yanniszark

**Comment by jlewi** [comment]
@yanniszark @kkasravi Did we decide who's going to work on this?

**Comment by yanniszark** [comment]
I got this assigned early in the slack conversation but I think I didn't mention it there, thought you had noticed.
I'm ok with either, I will have time to work on it tomorrow, alternatively @kkasravi can submit a pr and I will review tomorrow.

**Comment by kkasravi** [comment]
Thanks @yanniszark - I’ll have a PR hopefully in the AM

**Comment by jlewi** [comment]
This was partially fixed by #3673.

The default profile name ends up being truncated and having some random salt on it; 
e.g. `jlewi-at-google-com-3ddace`

I think the name should be deterministic. This makes it easier for users. In the rare case where an existing namespace already exists I think its fine not to create a new namespace. 

@gabrielwen will fix this in his upcoming PR #3640.

I'll leave this open until #3640 is merged and the name follows the desired conventions.

**Comment by gabrielwen** [comment]
@jlewi  I think it's fixed?
